cohortextractor/trino_utils.py

```diff
@@ -93,9 +93,17 @@ def trino_connection_params_from_url(url):
     parsed = urlparse(url)
     http_scheme = "https" if parsed.port == 443 else "http"
     parts = parsed.path.strip("/").split("/")
-    if len(parts) != 2 or not all(parts) or parsed.scheme != "trino":
+    # presto:// is now legacy and replaced with trino://
+    # presto:// is included for backwards compatibilty only and can be
+    # removed in future.
+    if (
+        len(parts) != 2
+        or not all(parts)
+        or parsed.scheme != "trino"
+        or parsed.scheme != "presto"
```

Comment:
I think you need
```py
or (parsed.scheme != "trinio" and parsed.scheme != "presto")
```
Or maybe neater to do it as
```py
or parsed.scheme not in ("trinio", "presto")
```

Reply:
I was going to use `not in`, then thought "why don't I make it easier to cleave off the change?" :man_facepalming: